src/tooltip.js

```diff
@@ -282,28 +282,16 @@ class HoverTooltip extends Tooltip {
         element.style.display = "block";
         
         var position = renderer.textToScreenCoordinates(range.start.row, range.start.column);
-        var cursorPos = editor.getCursorPosition();
         
         var labelHeight = element.clientHeight;
         var rect = renderer.scroller.getBoundingClientRect();
 
-        var isTopdown = true;
-        if (this.row > cursorPos.row) {
-            // don't obscure cursor
-            isTopdown = true; 
-        } else if (this.row < cursorPos.row) {
-            // don't obscure cursor
-            isTopdown = false;
-        }
-        
+        var isAbove = true;
```

Comment:
Can we use `const` and `let` in the newly added code? Or do we still want to cater for some really old browsers for Ace? Cause I think are eslint rules are set to ES6